doctr/models/recognition/sar/pytorch.py

```diff
@@ -21,8 +21,8 @@
 
 default_cfgs: Dict[str, Dict[str, Any]] = {
     'sar_resnet31': {
-        'mean': (.5, .5, .5),
-        'std': (1., 1., 1.),
+        'mean': (0.694, 0.695, 0.693),
+        'std': (0.299, 0.296, 0.301),
         'input_shape': (3, 32, 128),
         'vocab': VOCABS['legacy_french'],
```

Comment:
While we're at it, since we're gonna retrain it, I suggest switching to "french" vocab